frontend/src/views/Members.vue

```diff
@@ -43,24 +48,26 @@ SPDX-License-Identifier: Apache-2.0
           Access to resources within your project can be configured by assigning roles.
         </p>
       </v-card-text>
-      <v-list two-line subheader v-else>
-        <template v-for="(user, index) in sortedAndFilteredUserList">
-          <v-divider v-if="index !== 0" inset :key="index"></v-divider>
+      <v-data-table
+        v-else
+        :headers="visibleUserAccountTableHeaders"
+        :items="userList"
+        :footer-props="{ 'items-per-page-options': [5,10,20] }"
+        :options.sync="userAccountTableOptions"
```

Comment:
Here we use a sync property to distribute shared state across components. I think we should only use one possible solution for one type of problem. Why don't we share the state via the store. Even for reuse components we could have a path which directs the component to the correct store.state property. Also the localStorage handling should be done centrally in the store.

Reply:
we agreed to do this in a separate PR, see also https://github.com/gardener/dashboard/issues/905